Makefile

```diff
@@ -1,13 +1,13 @@
 COMMIT=$(shell git rev-parse --short HEAD)$(shell [[ $$(git status --porcelain --ignored) = "" ]] && echo -clean || echo -dirty)
 
 # all is the default target to build everything
-all: clean build sync e2e-bin logbridge
+all: clean build sync e2e-bin logbridge backup
```

Comment:
suggestion: maybe update to etcd-backup as there might be other backup tools in the future

Reply:
done